apps/backend/prisma/schema.prisma

```diff
@@ -96,3 +96,9 @@ model LeaderboardEntry {
   tapCount   Int?
   createdAt  DateTime @default(now())
 }
+
+model EdgeCityAttendance {
+  id        String    @id @default(uuid())
+  chipId    String
+  weeks     String[]
```

Comment:
I could make this more generic by changing to "Attendance" and adding a chipIssuer column, but I figured this was only relevant to UCL (when people go to Dev Con their tickets cover the entire time, right?).